tests/cds-videos/test_videos_transform_rules.py

```diff
@@ -0,0 +1,316 @@
+            {"name": "Esposito, William", "role": "Speaker", "affiliations": ["CERN"]},
+        ]
+        assert metadata["language"] == "en"
+
+
+def test_transform_description(datadir, base_app):
+    """Test that the description field `520` is correctly transformed."""
+    with base_app.app_context():
+        # Load test data
+        data = load_json(datadir, "lecture.json")
+
+        # Remove the 520 tag (description) from MARCXML
+        modified_data = data[0]
+        modified_data["record"][-1]["marcxml"] = remove_tag_from_marcxml(
+            modified_data["record"][-1]["marcxml"], "520"
+        )
+
+        dump = CDSRecordDump(data=modified_data, dojson_model=videos_migrator_marc21)
+        dump.prepare_revisions()
+        _, res = dump.latest_revision
+
+        # Ensure json_converted_record don't have the description
+        assert "description" not in res
+
+        # Transform record
+        record_entry = CDSToVideosRecordEntry()
+        metadata = record_entry._metadata(res)
+
+        # Ensure description exists and matches the title
+        assert metadata["description"] == metadata["title"]["title"]
+
+
+def test_transform_date(datadir, base_app):
+    """Test that the date field is correctly transformed."""
+    with base_app.app_context():
+        # Load test data
+        data = load_json(datadir, "lecture.json")
+
+        # Test case: Fail due to multiple dates
+        modified_data = data[0]
+        modified_data["record"][-1]["marcxml"] = add_tag_to_marcxml(
+            modified_data["record"][-1]["marcxml"], "518", {"d": "2025-02-06"}
+        )
+        dump = CDSRecordDump(data=modified_data, dojson_model=videos_migrator_marc21)
+        dump.prepare_revisions()
+        _, res = dump.latest_revision
+
+        # Transform record
+        record_entry = CDSToVideosRecordEntry()
+        with pytest.raises(UnexpectedValue):
+            record_entry._metadata(res)
+
+        # Test case: Fail due to missing dates
+        modified_data["record"][-1]["marcxml"] = remove_tag_from_marcxml(
+            modified_data["record"][-1]["marcxml"], "518"
+        )
+        modified_data["record"][-1]["marcxml"] = remove_tag_from_marcxml(
+            modified_data["record"][-1]["marcxml"], "269"
+        )
+
+        dump = CDSRecordDump(data=modified_data, dojson_model=videos_migrator_marc21)
+        dump.prepare_revisions()
+        _, res = dump.latest_revision
+
+        # Transform record
+        with pytest.raises(MissingRequiredField):
+            record_entry._metadata(res)
+
+
+def test_transform_contributor(datadir, base_app):
+    """Test that the date field is correctly transformed."""
+    with base_app.app_context():
+        # Load test data
+        data = load_json(datadir, "lecture.json")
+
+        # Test case: Fail due to missing contributor
+        modified_data = data[0]
+        modified_data["record"][-1]["marcxml"] = remove_tag_from_marcxml(
```

Comment:
@zubeydecivelek suggestion for your next PR
```suggestion
        record_marcxml = modified_data["record"][-1]["marcxml"] 
       record_marcxml= remove_tag_from_marcxml(
```